stage #4/module #5. AWS/theory/ec2.md

```diff
@@ -0,0 +1,171 @@
+# Amazon EC2
+
+## Recommended timeline
+
+The recommended timeline for the whole EC2 module is approximately 6 hours.
```

Comment:
There is a file called aws_task.md introduced there https://github.com/mjc-school/MJC-School/pull/153
Please remove the timline per each service since a single timeline given for the whole module

Reply:
The timelines have been removed